client/state/account-recovery/test/reducer.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ACCOUNT_RECOVERY_SETTINGS_FETCH,
+	ACCOUNT_RECOVERY_SETTINGS_FETCH_SUCCESS,
+	ACCOUNT_RECOVERY_SETTINGS_FETCH_FAILED,
+} from 'state/action-types';
+
+import reducer from '../reducer';
+
+describe( '#account-recovery/isFetchingSettings reducer :', () => {
+	it( 'should set isFetchingSettings flag.', () => {
+		const state = reducer( undefined, {
+			type: ACCOUNT_RECOVERY_SETTINGS_FETCH,
+		} );
+
+		assert.isTrue( state.isFetchingSettings );
+	} );
+
+	it( 'should unset isFetchingSettings flag on success.', () => {
+		const state = reducer( undefined, {
+			type: ACCOUNT_RECOVERY_SETTINGS_FETCH_SUCCESS,
+			settings: {
+				email: '',
+				email_validated: true,
+				phone: {},
+				phone_validated: true,
```

Comment:
I see your point now. It might be not the best name of it, but I have no better idea yet and it would require to patch the endpoint. Is it okay to you to keep it this way?